internal/discord/handlers.go

```diff
@@ -0,0 +1,251 @@
+		for _, command := range commandList {
+			fmt.Printf("\t%v\n", command.Name)
+		}
+		if err != nil {
+			fmt.Printf("Could not add some commands: %v \n", err)
+		}
+	}
+}
+
+var KickCommand = &discordgo.ApplicationCommand{
+	Name:                     "kick",
+	DefaultMemberPermissions: &kickPermission,
+	Description:              "Kick the specified user and notify the user why via DMs.",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Type:        discordgo.ApplicationCommandOptionUser,
+			Name:        "user",
+			Description: "User being kicked",
+			Required:    true,
+		},
+		{
+			Type:        discordgo.ApplicationCommandOptionString,
+			Name:        "reason",
+			Description: "Reason for kick",
+			Required:    false,
+		},
+	},
+}
+
+var MuteCommand = &discordgo.ApplicationCommand{
+	Name:                     "mute",
+	DefaultMemberPermissions: &mutePermission,
+	Description:              "Mute the specified user.",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Type:        discordgo.ApplicationCommandOptionUser,
+			Name:        "user",
+			Description: "User being muted",
+			Required:    true,
+		},
+		{
+			Type:        discordgo.ApplicationCommandOptionInteger,
+			Name:        "duration",
+			Description: "Duration in minutes",
+			Required:    true,
+		},
+		{
+			Type:        discordgo.ApplicationCommandOptionString,
+			Name:        "reason",
+			Description: "Reason for mute",
+			Required:    false,
+		},
+	},
+}
+
+var UnmuteCommand = &discordgo.ApplicationCommand{
+	Name:                     "unmute",
+	DefaultMemberPermissions: &mutePermission,
+	Description:              "Unmute the specified user.",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Type:        discordgo.ApplicationCommandOptionUser,
+			Name:        "user",
+			Description: "User being muted",
+			Required:    true,
+		},
+		{
+			Type:        discordgo.ApplicationCommandOptionString,
+			Name:        "reason",
+			Description: "Reason for mute",
+			Required:    false,
+		},
+	},
+}
+
+var BanCommand = &discordgo.ApplicationCommand{
+	Name:                     "ban",
+	DefaultMemberPermissions: &banPermission,
+	Description:              "Ban the specified user.",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Type:        discordgo.ApplicationCommandOptionUser,
+			Name:        "user",
+			Description: "User being banned",
+			Required:    true,
+		},
+		{
+			Type:        discordgo.ApplicationCommandOptionString,
+			Name:        "reason",
+			Description: "Reason for ban",
+			Required:    false,
+		},
+	},
+}
+
+var UnbanCommand = &discordgo.ApplicationCommand{
+	Name:                     "unban",
+	DefaultMemberPermissions: &banPermission,
+	Description:              "Unban the specified user.",
+	Options: []*discordgo.ApplicationCommandOption{
+		{
+			Type:        discordgo.ApplicationCommandOptionUser,
+			Name:        "user",
+			Description: "User being unbanned",
```

Comment:
might need to be more specific in unbans since the user is no longer in the server. Might need the actual discord ID

Reply:
Adding a user ID in the `user` field works too!

Reply:
just to be clear is this only a discord id itself or can it also be a nickname?

Reply:
we should specify in the description that it's discord ID only